### PR TITLE
begin docs for pillows

### PR DESCRIPTION
**Pull request by emord**
Some basic overview docs @kaapstorm

### CODE REVIEW

File: docs/pillows.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+1. Change Feed
+2. Checkpoint
+3. Processor(s)
+4. Change Event Handler
+
+Change Feed
+-----------
+
+Change feeds are documented `here <change_feed>`_.
```
Comment (kaapstorm) [inline docs/pillows.rst]: Possibly needs to be ``` `here <Change Feeds>`_``` to match the title "Change Feeds".

**Review by kaapstorm — APPROVED** [review]

**Comment by kaapstorm** [comment]
:+1: Great docs!

**Review by benrudolph — APPROVED** [review]
nice

**Comment by emord** [comment]
Just docs so going to self merge

File: docs/pillows.rst [hunk 2]
```diff
@@ -107,3 +107,21 @@ to process changes that have already been processed. This usually happens when
 a couch node fails over to another. If this occurs, stop the pillow, wait for
 confirmation that the couch nodes are up, and fix the checkpoint using:
 `./manage.py fix_checkpoint_after_rewind <pillow_name>`
+
+Problem with checkpoint for pillow name: First available topic offset for topic is num1 but needed num2
```
Comment (emord) [inline docs/pillows.rst]: @biyeun documented as promised

**Pull request merged**